test_scripts/Polices/build_options/026_ATF_P_TC_PTU_PM_Sets_Status_UPDATE_NEEDED_HTTP.lua

```diff
@@ -127,17 +128,18 @@ function Test:RAI_PTU()
         function()
           log("SDL->MOB: N: OnPermissionsChange")
         end)
-      :Times(1)
     end)
+
+  commonTestCases:DelayedExp(65000) -- wait exchange_after_x_seconds(t0) + tollerance 5 sec
 end
 
 Test["Starting waiting cycle [" .. attempts * 5 .. "] sec"] = function() end
 
-for i = 1, attempts do
-  Test["Waiting " .. i * 5 .. " sec"] = function()
-    os.execute("sleep 5")
-  end
-end
+-- for i = 1, attempts do
```

Comment:
@dboltovskyi, according to me verifications should have stable verification in all scripts. What about if in future additional check for something else is added and during this check notification is come, but time for waiting is elapsed? I think that appropriate way is to have expectations only inside of step(function), not during the whole script.